src/chains/ethereum_1/ethAction.ts

```diff
@@ -85,8 +84,13 @@ export class EthereumActionHelper {
       contract,
     } = actionInput
 
+    const gasPriceInWei =
```

Comment:
Make a helper function 

  gweiToWeiString(value: EthereumMultiValue): string {
    // if string .. convert to number
    // if number, toWei()
    // return string
  }